config/nonprod.tfvars

```diff
@@ -0,0 +1,13 @@
+env = "nonprod"
+subusers = [
+    {
```

Comment:
this config should be abstracted away so teams don't need to know the sendgrid bits behind the scenes.

I would have 'name' for the account, and that name should always be prepended with hmcts- (in the terraform code to avoid clashed with other organisations.

ips shouldn't need to be specified here, but we could allow them to be overridden here

ideally email should be unique for each account and forward on to us, but I think that would be quite painful so we should probably just use DTSPlatformOps@HMCTS.NET

Reply:
agreed.  only name and an optional ip now left in the config.